tests/test_resources/test_modules/test_folder.py

```diff
@@ -17,6 +17,19 @@ def fs_str_rh_fn(folder):
 
 
 class TestFolder(tests.test_resources.test_resource.TestResource):
+
+    MAP_FIXTURES = {"resource": "folder"}
+
+    UNIT = {"folder": ["s3_folder", "gcs_folder"]}
+    LOCAL = {"folder": ["s3_folder", "gcs_folder"]}
+    MINIMAL = {"folder": ["cluster_folder", "s3_folder", "gcs_folder"]}
+    THOROUGH = {
```

Comment:
thorough and maximal should include cases too; otherwise passing in an empty list will cause it to not parameterize the folder argument at all in this level